### PR TITLE
fix(example): import styles as ESModules

### PR DESCRIPTION
**Pull request by mottox2**
## Description

This PR fixed a broken using-css-modules example.
In Gatsby v3, CSS Modules are imported as ESModules.

### CODE REVIEW

File: examples/using-css-modules/src/styles/another-page.module.css [hunk 1]
```diff
@@ -8,9 +8,10 @@
   font-size: 10rem;
   line-height: 1.5;
   text-decoration: none;
-  &:hover {
-    text-decoration: underline;
-  }
+}
+
+.link:hover {
```
Comment (wardpeet) [inline examples/using-css-modules/src/styles/another-page.module.css:13]: is this change necessary?

**Review by wardpeet — COMMENTED** [review]
Thanks for upgrading! 🎉 Added one question

File: examples/using-css-modules/src/styles/another-page.module.css (hunk 1 above)
Comment (mottox2) [reply examples/using-css-modules/src/styles/another-page.module.css:13]: Yes. In the previous code, the style was not enabled.

**Review by wardpeet — APPROVED** [review]
Thank you! :tada:
